engine/app/lib/citizens_advice_components/elements/date_input.rb

```diff
@@ -21,7 +21,11 @@ def render
       private
 
       def legend_html
-        tag.legend(class: "cads-form-field__label") { safe_join([label, " ", optional_html]) }
+        if options[:page_heading].present?
+          tag.h1(class: "cads-page-title") { label }
```

Comment:
I think this `h1` needs to be nested within a `legend` tag for it to be the same as the Design System (see [here](https://github.com/citizensadvice/design-system/blob/baa16a5af029e2f8c62bb2dc149e148ee8918ee2/engine/app/components/citizens_advice_components/date_input.html.erb#L5))